### ISSUE TITLE
inconsistent handling ../\\ as a filepath

### ISSUE THREAD

**Issue by rbtcollins** [nushell/nushell#1688]
**Describe the bug**
I made a file called `\` (`\\` with escaping) by mistake, and can cat it but not rm it within nushell.

**To Reproduce**
1. outside of nushell, touch `../\\` 
2. inside of nushell, `cat ../\\` works
3. inside of nushell  `rm ../\\` fails.

```
❯ cat ../\\
pick 0c36d58 ...
...
❯ rm ../\\
error: No valid paths
- shell:1:8
1 | rm ../\\
  | ^^ no valid paths
> rm ../\
deleted /full/path/../\
```

**Expected behavior**
Cope with the path escaping consistently. Tab completion for instance was giving me the `\\` escaped version which worked with cat but not with rm.
**Configuration (please complete the following information):**
 - OS: Linux
 - Version 0.13
 - Optional features (if any): stable

**Comment by thegedge** [comment]
The reason `rm` fails is because `rm` is a builtin command, whereas `cat` is an external. Does `rm "../\"` work?

Regardless, this still feels awkward, and we should bring these closer together.

**Comment by rbtcollins** [comment]
Yes - its the last line in my transscript. The issue was that tab completion for `rm` tab completes the way it needs to for `cat`.

**Comment by thegedge** [comment]
Ah, my apologies for completely missing that 🙈 🤦 

We haven't completely decided on escaping semantics in nushell. One way we could deal with this, though, is by escaping things before shelling out. That way these two should work:

- `rm ../\`
- `cat ../\`

**Comment by rbtcollins** [comment]
I'm not sure that the stale system is going to help here, should I answer every X days to say that this is still a thing? Or will the core team be reviewing the bugs to follow up on them? I can't just remove the stale label because of permissions.

**Comment by rbtcollins** [comment]
comment below

**Comment by jntrnr** [comment]
In Nushell, to have direct control over what the filename is, you'd use backticks, like:

```
> rm `/`
> rm `\`
```

And so on. `cat` is an external command and may process the path differently than the internal commands. But yeah, as a general rule for any internal command, the backticks should allow you to state exactly what the file path should be.

In your example, `rm ../\` works and as Nushell doesn't use path escaping, this is expected.
